### PR TITLE
Find openal build flags using pkg-config

### PR DESCRIPTION
**Pull request by thibautbuchert**
Use the environment variable added in https://github.com/razterizer/Core/pull/6 to figure out compiler flags for openal using pkg-config.

I have removed all the `additional_flags`, hopefully `pkg-config` will find them even on MacOS.

### CODE REVIEW

**Comment by thibautbuchert** [comment]
@razterizer I am really curious to know whether this will work on MacOS.

File: Pilot_Episode/build.sh
Comment (thibautbuchert) [inline Pilot_Episode/build.sh:1]: I need to change the commit hash for Core in the dependencies before this PR can be merged (and this can only be done once https://github.com/razterizer/Core/pull/6 is merged)

**Review by razterizer — APPROVED** [review]

**Pull request merged**
